src/denish/Assignment19.java

```diff
@@ -0,0 +1,29 @@
+package denish;
+
+public class Assignment19 {
+	
+	void printLength(String input) {
+		
+		for(int index=input.length()-1;index>=0;index--) {
+			System.out.println(index+ "->"+input.charAt(index));
+		
+	}
+}
+	void process(String input){
+		int count=0;
+		char ch='t';
```

Comment:
instead of making this static, you can create this generic, means user will give you character and you have to tell its frequency. here, your code is only working for character 't'